blackbelt/config.py

```diff
@@ -23,6 +27,8 @@
     }
 # default config, overwrite/modularize
 
-config['trello']['work_board_id'] = '1KsoiV9e'
+if not config['trello']['work_board_id']:
+    config['trello']['work_board_id'] = '1KsoiV9e'
```

Comment:
Magic string? Could this point to a constant that is more understandable? Presume this is DEFAULT_BOARD_ID.